include/spl/sys/debug.h

```diff
@@ -107,13 +144,13 @@ void spl_dumpstack(void);
  */
 #else
 
-#define	ASSERT(cond)		VERIFY(cond)
+#define	ASSERT3B	VERIFY3B
+#define	ASSERT3S	VERIFY3S
+#define	ASSERT3U	VERIFY3U
+#define	ASSERT3P	VERIFY3P
+#define	ASSERT0		VERIFY0
+#define	ASSERT		VERIFY
 #define	ASSERTV(x)		x
-#define	ASSERT3B(x,y,z)		VERIFY3B(x, y, z)
-#define	ASSERT3S(x,y,z)		VERIFY3S(x, y, z)
-#define	ASSERT3U(x,y,z)		VERIFY3U(x, y, z)
-#define	ASSERT3P(x,y,z)		VERIFY3P(x, y, z)
-#define	ASSERT0(x)		VERIFY0(x)
```

Comment:
The userspace equivalents of these macros are in `lib/libspl/include/assert.h` and will need to be updated too.